### PR TITLE
Feature/htslib sif

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- htslib as sif

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [x] Code review
- [x] New code is executed and covered by tests
- [x] Tests pass

### CODE REVIEW

File: containers/Singularity.htslib-1.9 [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Bootstrap: docker
+From: ubuntu:xenial
+
+%help
+    This is a container for installing bcftools, bgzip, htslib, samtools and tabix
```
Comment (jemten) [inline containers/Singularity.htslib-1.9]: Nice to bundle the tools! 👍

Comment (henrikstranneheim) [reply containers/Singularity.htslib-1.9]: Yeah, they seem to belong together

**Comment by henrikstranneheim** [comment]
Sorry I missed the comment about the CLI. What did you mean?

**Comment by jemten** [comment]
In the install cli you have the option to select which programs that are to be installed. I think that bcftools and samtools are available options but if you supply them now it won't work. Better to remove them and only have htslib which will install all of them. 
But no worries, we can take that later

**Comment by henrikstranneheim** [comment]
Ah, yes of course now I see what you mean. I will do another PR

**Pull request merged**